README.md

```diff
@@ -363,77 +366,77 @@ Testing:
+|                  C++/g++ | 0.132<sub>±0.000</sub> |    3.54<sub>±00.07</sub> + 85.27<sub>±00.26</sub> |   5.36<sub>±00.04</sub> |
+|              C++/clang++ | 0.143<sub>±0.000</sub> |    3.08<sub>±00.08</sub> + 75.41<sub>±00.00</sub> |   5.63<sub>±00.05</sub> |
+|                  V/clang | 0.148<sub>±0.000</sub> |   0.72<sub>±00.00</sub> + 262.80<sub>±00.57</sub> |   6.15<sub>±00.06</sub> |
+|                    V/gcc | 0.150<sub>±0.000</sub> |   0.71<sub>±00.06</sub> + 259.28<sub>±00.48</sub> |   6.28<sub>±00.04</sub> |
+|                     Java | 0.155<sub>±0.002</sub> |  36.96<sub>±00.11</sub> + 144.23<sub>±05.61</sub> |   8.63<sub>±00.15</sub> |
+|                  Node.js | 0.256<sub>±0.003</sub> |  36.96<sub>±00.11</sub> + 181.19<sub>±02.81</sub> |  12.80<sub>±00.06</sub> |
+|               Lua/luajit | 0.337<sub>±0.001</sub> |   2.49<sub>±00.03</sub> + 156.75<sub>±00.38</sub> |  12.87<sub>±00.03</sub> |
+|                    Scala | 0.391<sub>±0.005</sub> |  66.10<sub>±00.30</sub> + 258.71<sub>±02.41</sub> |  22.50<sub>±00.24</sub> |
+|                    Julia | 0.555<sub>±0.002</sub> | 230.82<sub>±00.22</sub> + 327.93<sub>±01.74</sub> |  23.58<sub>±00.12</sub> |
+|              Python/pypy | 0.880<sub>±0.002</sub> |  64.77<sub>±00.07</sub> + 249.36<sub>±00.15</sub> |  34.55<sub>±00.17</sub> |
+|             Ruby (--jit) | 1.443<sub>±0.005</sub> | 270.96<sub>±00.02</sub> + 146.80<sub>±00.04</sub> |  58.79<sub>±00.46</sub> |
+|                      Lua | 1.480<sub>±0.005</sub> |   2.32<sub>±00.01</sub> + 283.97<sub>±00.87</sub> |  57.87<sub>±00.19</sub> |
+|         Ruby/truffleruby | 1.563<sub>±0.012</sub> | 287.18<sub>±00.42</sub> + 334.66<sub>±24.08</sub> |  80.19<sub>±00.54</sub> |
+|               Ruby/jruby | 1.883<sub>±0.026</sub> | 182.58<sub>±00.29</sub> + 370.42<sub>±55.17</sub> | 101.51<sub>±01.01</sub> |
+| Ruby/truffleruby (--jvm) | 1.895<sub>±0.018</sub> | 348.72<sub>±09.17</sub> + 526.76<sub>±97.23</sub> | 123.08<sub>±01.27</sub> |
+|                     Ruby | 2.032<sub>±0.004</sub> |  14.36<sub>±00.02</sub> + 146.75<sub>±00.00</sub> |  82.09<sub>±00.60</sub> |
+|                   Python | 4.806<sub>±0.058</sub> |  10.41<sub>±00.06</sub> + 234.92<sub>±00.64</sub> | 190.06<sub>±02.19</sub> |
 
 # Tests Execution
 
 ## Environment
 
-CPU: Intel(R) Core(TM) i7-10710U
+CPU: Intel(R) Xeon(R) E-2324G
 
 Base Docker image: Debian GNU/Linux bookworm/sid
 
 | Language         | Version                         |
 | ---------------- | ------------------------------- |
-| .NET Core        | 6.0.201                         |
-| C#/.NET Core     | 4.1.0-5.22116.13 (dbffaa4a)     |
-| C#/Mono          | 6.12.0.122                      |
+| .NET Core        | 6.0.203                         |
+| C#/.NET Core     | 4.1.0-5.22128.4 (5d10d428)      |
+| C#/Mono          | 6.8.0.105                       |
```

Comment:
mono version seems downgraded?

Reply:
Yes, mono custom Debian repository gave me few issues, so I've decided to not use it anymore.